<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hangman</class>
 <widget class="QDialog" name="Hangman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hangman</string>
  </property>
  <widget class="QPushButton" name="A">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="C">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="D">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="E">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="F">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="G">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="J">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="I">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="H">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="K">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="S">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="R">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Q">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>500</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="O">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>500</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="M">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="L">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Y">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>640</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="X">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>640</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="W">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>640</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="V">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>640</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="U">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Z">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>640</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>10</y>
     <width>1161</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>A</zorder>
  <zorder>B</zorder>
  <zorder>C</zorder>
  <zorder>D</zorder>
  <zorder>E</zorder>
  <zorder>F</zorder>
  <zorder>G</zorder>
  <zorder>J</zorder>
  <zorder>I</zorder>
  <zorder>H</zorder>
  <zorder>K</zorder>
  <zorder>S</zorder>
  <zorder>R</zorder>
  <zorder>Q</zorder>
  <zorder>P</zorder>
  <zorder>O</zorder>
  <zorder>N</zorder>
  <zorder>M</zorder>
  <zorder>L</zorder>
  <zorder>Y</zorder>
  <zorder>X</zorder>
  <zorder>W</zorder>
  <zorder>V</zorder>
  <zorder>U</zorder>
  <zorder>T</zorder>
  <zorder>Z</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>lineEdit_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
